packages/editor/api-report.md

```diff
@@ -2096,57 +2075,6 @@ export type TLAnyBindingUtilConstructor = TLBindingUtilConstructor<any>;
 // @public (undocumented)
 export type TLAnyShapeUtilConstructor = TLShapeUtilConstructor<any>;
 
-// @public (undocumented)
-export interface TLArcInfo {
-    // (undocumented)
-    center: VecLike;
-    // (undocumented)
-    largeArcFlag: number;
-    // (undocumented)
-    length: number;
-    // (undocumented)
-    radius: number;
-    // (undocumented)
-    size: number;
-    // (undocumented)
-    sweepFlag: number;
-}
-
-// @public (undocumented)
-export interface TLArrowBindings {
-    // (undocumented)
-    end: TLArrowBinding | undefined;
-    // (undocumented)
-    start: TLArrowBinding | undefined;
-}
-
-// @public (undocumented)
-export type TLArrowInfo = {
```

Comment:
I think these types were supposed to remain public in the `'tldraw'` package but it looks like they're not being exported from the index there?

Reply:
ahh good catch, thank you!